docs/faq.md

```diff
@@ -0,0 +1,31 @@
+# AKS-Engine FAQ
+
+This page provides help with the most common questions about AKS-Engine.
+
+### What's the Difference Between AKS and AKS-Engine?
+
+Azure Kubernetes Service ([AKS][]) is a Microsoft Azure service that supports fully managed Kubernetes clusters. [AKS-Engine][] is an Azure open source project that creates Kubernetes clusters with your custom requirements. AKS uses AKS-Engine internally, but they are not the same.
+
+AKS clusters can be created in the Azure portal or with `az aks create` in the [Azure command-line tool][]. AKS-Engine clusters can be created with `aks-engine deploy` in the AKS-Engine command-line tool, or by generating the ARM templates with `aks-engine generate` and deploying them as a separate step.
+
+### What's the Difference Between `acs-engine` and `aks-engine`?
+
+AKS-Engine is the next version of the ACS-Engine project. AKS-Engine supports current and future versions of [Kubernetes][], while ACS-Engine also supported the Docker Swarm and Mesos DC/OS container orchestrators.
+
+### Is ACS-Engine Still Active?
+
+Security fixes and other critical changes will still go into ACS-Engine and patch releases will be made as needed. But in general, improvements and releases with new features will only be made in AKS-Engine.
```

Comment:
We should emphasize that aks-engine is the continuity of acs-engine and is backwards compatible therefore any bug fixes that were usually released in a new acs-engine release will be released in a new aks-engine release.

Reply:
I agree with the sentiment but I'm having a hard time wording that cleanly. How about this?

> Security fixes and other critical changes will go into both the ACS-Engine and AKS-Engine projects, and patch releases will be made as needed. But in general, improvements and releases with new features will only be made in AKS-Engine.

> AKS-Engine is the continuation of ACS-Engine and is backward-compatible. The feature or fix you wanted to see in ACS-Engine should be equally relevant to AKS-Engine.

Reply:
I guess I don't like the first sentence "Security fixes and other critical changes will go into both the ACS-Engine and AKS-Engine projects, and patch releases will be made as needed", it sounds like we are going to keep releasing acs-engine after aks-engine is released which I understood we weren't going to do. Any security fix that's needed can be released in aks-engine since that will be considered the continuation of acs-engine.

Reply:
I see what you mean. And I'm not entirely sure we intend to do acs-engine releases even in this case, if we can point to an aks-engine release that contains the fix... I'll make another attempt at rewording this and de-emphasize the potential for more acs-engine releases.

Reply:
@CecileRobertMichon I replaced it with this, let me know what you think:

> No further development or releases in ACS-Engine are planned. AKS-Engine is a backward-compatible continuation of ACS-Engine, so all fixes and new features will target AKS-Engine.

Reply:
👍